libs/island-ui/core/src/lib/ActionCard/ActionCard.tsx

```diff
@@ -172,7 +200,10 @@ export const ActionCard: React.FC<ActionCardProps> = ({
           <Text variant="small">{date}</Text>
         </Box>
 
-        {!eyebrow && renderTag()}
+        <Inline alignY="center" space={1}>
+          {!eyebrow && renderTag()}
+          {!eyebrow && renderDelete()}
```

Comment:
This is confusing, render delete when eyebrow is not defined? But I guess that comes more down to the structure of this component

Sidenote: Rendering React jsx in this way avoides React's optimization services and should generally be avoided.